be/src/olap/rowset/rowset_writer.h

```diff
@@ -95,6 +96,24 @@ class RowsetWriter {
         return Status::NotSupported("RowsetWriter does not support create_file_writer");
     }
 
+    virtual Status create_inverted_index_file_writer(
+            uint32_t segment_id, InvertedIndexFileWriterPtr* index_file_writer) {
+        std::string segment_prefix {InvertedIndexDescriptor::get_index_file_path_prefix(
+                _context.segment_path(segment_id))};
+        // Create file writer for the inverted index format v2.
+        io::FileWriterPtr idx_file_v2_ptr;
+        if (_context.tablet_schema->get_inverted_index_storage_format() !=
+            InvertedIndexStorageFormatPB::V1) {
+            RETURN_IF_ERROR(
+                    create_file_writer(segment_id, idx_file_v2_ptr, FileType::INVERTED_INDEX_FILE));
+        }
+        *index_file_writer = std::make_unique<InvertedIndexFileWriter>(
+                _context.fs(), segment_prefix, _context.rowset_id.to_string(), segment_id,
+                _context.tablet_schema->get_inverted_index_storage_format(),
+                std::move(idx_file_v2_ptr));
```

Comment:
InvertedIndexFileWriter's constructor already has std::move, double move?

Reply:
`std::move()` only marks this value as an rvalue.